hswidget/dt1widget/widget.go

```diff
@@ -579,13 +580,21 @@ func (p *widget) SetTileGroup(tileGroup int32) {
 }
 
 func (p *widget) makeSubtileFlags(state *widgetState, tile *d2dt1.Tile) giu.Layout {
+	sf := state.subtileFlag
```

Comment:
you don't need to do that, just use `state.subtileFlag` below

Reply:
True! I'll fix this. This means that the OnChange func can also be removed making the giu.Combo a single line